File: 백준/Bronze/13458. 시험 감독/시험 감독.js
const input= require('fs').readFileSync('/dev/stdin').toString().trim().split('\n')
const n = Number(input[0])
const A = input[1].split(' ').map(Number)
const [b,c] = input[2].split(' ').map(Number)
let num = n
for(let a of A){
    let start = a - b
    if(start>0){
        num += Math.ceil(start/c)
    }
}
console.log(num)

File: 백준/Bronze/13458. 시험 감독/시험 감독.py
import sys,math
input = sys.stdin.readline
n = int(input())
a = list(map(int, input().split()))
b, c, = map(int, input().split())
result = n
for i in a:
    num = i-b
    if num > 0:
        result += math.ceil(num/c)
print(result)
